src/modal/sign-txn/_pera-wallet-sign-txn-modal.scss

```diff
@@ -0,0 +1,17 @@
+.pera-wallet-sign-txn-modal.pera-wallet-modal {
+  .pera-wallet-modal__body {
+    width: 480px;
+    height: 640px;
+
+    #pera-wallet-iframe {
+      position: absolute;
```

Comment:
Is there any specific reason to use absolute positioning? I'd go without it

Reply:
I'm using this because we have background with lines in modal. I do it like this so that the modal takes up the whole space

Reply:
Can we have a separate modal style for this purpose?